core/common/src/main/java/alluxio/underfs/UnderFileSystem.java

```diff
@@ -367,17 +318,17 @@ public OutputStream create(String path, CreateOptions options) throws IOExceptio
    * @return The list of locations
    * @throws IOException if a non-Alluxio error occurs
    */
-  public abstract List<String> getFileLocations(String path) throws IOException;
+  List<String> getFileLocations(String path) throws IOException;
 
   /**
-   * Gets the list of locations of the indicated path given its offset.
+   * Gets the list of locations of the indicated path given options.
    *
    * @param path the file name
-   * @param offset the offset in bytes
+   * @param options including offset within the file
```

Comment:
it is fine to just say "method options" so that the javadoc does not need to be update / become stale when new options are added